ChangeLog.md

```diff
@@ -1,22 +1,27 @@
 # Changelog
```

Comment:
Let's also include James's fixes for the server deployments to this release, and let's release it first thing tomorrow!!!